### ISSUE TITLE
Remove mention of threadable.init from the threading chapter

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2212]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#2212|
|Type|enhancement|
|Created|2006-11-01 20:05:34Z|

threading.xhtml still mentions that you should call threadable.init(1) as the first thing in your threaded program. This hasn't been necessary for a super mega long time. 

Attachments:

* [init-sux.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ebc/ebc1e93c8b301a465e37bf5c64319c4ab81e119b/88a38dee676def5fd1c46cbaf59dd1b65b0fcf10.patch) (1081 bytes) - added by radix on 2006-11-07 21:55:10Z - hooray

<details><summary>Searchable metadata</summary>

```
trac-id__2212 2212
type__enhancement enhancement
reporter__radix radix
priority__highest highest
milestone__Core_2_5 Core-2.5
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1162411534000000 1162411534000000
changetime__1164297945000000 1164297945000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Why Twisted 2.5?  This is not a regression, so it isn't release blocking.

Are we using the milestone for release blocking tickets or just stuff we'd like to see get done?

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix set owner to PenguinOfDoom|
|-|-|

winner is me

**Comment by twisted-trac** [comment]
|<img alt="PenguinOfDoom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|PenguinOfDoom set owner to @radix|
|-|-|

APPROVED. RELEASE THE COMMIT TORPEDOES.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_7"></a>|@jml commented|
|-|-|

This is reviewed and blocking the 2.5 release. Making highest priority.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun commented|
|-|-|

Note that radix attached a new patch after PenguinOfDoom's review (addressing some comments I made on IRC).

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun set owner to @exarkun|
|-|-|

New patch looks good.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_10"></a>|@exarkun set owner to @radix|
|-|-|

trac, you piece of shit.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_11"></a>|@radix set status to `closed`|
|-|-|

(In [18768]) Remove mentions of threadable.init from the documentation

Author: radix
Reviewer: pahan, exarkun
Fixes [#2212](https://github.com/twisted/twisted/issues/2212)

threadable.init has been unnecessary for ages, but was
still mentioned in the threading chapter. No longer!

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
